pkg/test/framework/components/echo/deployment/builder.go

```diff
@@ -110,7 +110,7 @@ type builder struct {
 	// Only the first per-cluster entry for a given config should have a non-nil ref.
 	refs map[cluster.Kind][]*echo.Instance
 	// namespaces caches namespaces by their prefix; used for converting Static namespace from configs into actual
-	// namesapces
+	// namespaces
```

Comment:
Extra change? Haven't seen this in the original PR.

Reply:
removed